x/launch/keeper/msg_create_chain.go

```diff
@@ -13,6 +13,16 @@ import (
 func (k msgServer) CreateChain(goCtx context.Context, msg *types.MsgCreateChain) (*types.MsgCreateChainResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
+	// check if the metadata length is valid
+	maxMetadataLength := k.campaignKeeper.GetParams(ctx).MaxMetadataLength
```

Comment:
This is not the same type of metadata for chains

We need to implement another param in the `launch` module for the chain metadata that can be updated independently